cli/cmd/resource.go

```diff
@@ -31,17 +29,27 @@ func init() {
 }
 
 func runResource(cmd *cobra.Command, args []string) {
-	URL := viper.GetString("zabbix.url")
+	URL := viper.GetString("zabbix.URL")
 	key := viper.GetString("zabbix.api_key")
-	a := zabbix.NewAPI(URL, key)
+	a, err := zabbix.NewAPI(URL, key, "", "")
```

Comment:
Perhaps the exposed API for `NewAPI` should be declared with these 2 parameters.
I'ld rename NewAPI to `CreateClientUsingAPIKey`